tap_pendo/streams.py

```diff
@@ -959,7 +852,6 @@ class Visitors(LazyAggregationStream):
     replication_method = "INCREMENTAL"
     replication_key = "lastupdated"
     key_properties = ["visitor_id"]
-    method = "POST"
 
     def get_endpoint(self):
```

Comment:
@namrata270998 Is this function being used anywhere?

Reply:
No, removed it